### PR TITLE
Add recipe for pyinaturalist-convert

### PR DESCRIPTION
**Pull request by JWCook**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/main/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by JWCook** [comment]
@conda-forge/help-python
Ready for review!

File: recipes/pyinaturalist-convert/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+test:
+  imports:
+    - pyinaturalist_convert
+  requires:
+    - pip
+  commands:
+    - pip check
+
+about:
+  home: https://github.com/JWCook/pyinaturalist_convert
```
Comment (dopplershift) [inline recipes/pyinaturalist-convert/meta.yaml:48]: Typo in url:
```suggestion
  home: https://github.com/JWCook/pyinaturalist-convert
```

File: recipes/pyinaturalist-convert/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  summary: Convert iNaturalist observation data to and from multiple formats
+  license: MIT
+  license_file: LICENSE
+  doc_url: https://pyinaturalist.readthedocs.io/
+  dev_url: https://github.com/JWCook/pyinaturalist_convert
```
Comment (dopplershift) [inline recipes/pyinaturalist-convert/meta.yaml:53]: Here too:
```suggestion
  dev_url: https://github.com/JWCook/pyinaturalist-convert
```

File: recipes/pyinaturalist-convert/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+{% set hash_val = "7c02565c8100d9bc05b63872858ec7c13f88f441c3d64c702e29e6991efc3cbd" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ hash_val }}
```
Comment (dopplershift) [inline recipes/pyinaturalist-convert/meta.yaml:12]: These days we usually just put the hash directly here rather than templating it.

**Review by dopplershift — CHANGES_REQUESTED** [review]
Just need to fix the typos in the URLs, and might as well remove the templating of the hash while you're there. Otherwise, this is good to go with minor tweaks.

**Review by dopplershift — APPROVED** [review]

**Comment by JWCook** [comment]
Fixed! Thanks for reviewing. For some reason the OSX build has been stuck for almost 2 hours. Going to try restarting it.

**Comment by JWCook** [comment]
@dopplershift There we go, it finished this time. Ready to merge.

**Pull request merged**